### ISSUE TITLE
Underground backgrounds don't tile properly when a resource pack that changes them is enabled

### ISSUE THREAD

**Issue by QuibopWon** [bluemagic123/tModLoader#3896]
### Version

Latest 1.4.4-stable

### OS

Windows

### Platform

Steam

### This bug affects

Gameplay as a Player

### Description

![weird background bug 2](https://github.com/tModLoader/tModLoader/assets/85258495/d2887a6b-f878-4114-882a-a162ee2529ff)
it's hard to see because of how choppy the gif is (it's easier to see when i slow down), but the background sprites for the underground render over eachother instead of tiling properly
also, yes, the background sprite should tile perfectly
![image](https://github.com/tModLoader/tModLoader/assets/85258495/db054ff2-ad11-46f6-a600-5001cd4fff47)
i don't know if this happens in vanilla, but for whatever reason this only happens if the resource pack i'm using is on

### Log File

[Natives.log](https://github.com/tModLoader/tModLoader/files/13061767/Natives.log)
[client.log](https://github.com/tModLoader/tModLoader/files/13061768/client.log)
[environment-client.log](https://github.com/tModLoader/tModLoader/files/13061769/environment-client.log)
[Launch.log](https://github.com/tModLoader/tModLoader/files/13061770/Launch.log)

### Steps to reproduce

1. enable resource pack that changes underground backgrounds
2. make sure parallax is on
3. look at the background

### Expected Behavior

![image](https://github.com/tModLoader/tModLoader/assets/85258495/7149657f-1dfa-46a6-a3c5-628a19645e14)

### Actual Behavior

![image](https://github.com/tModLoader/tModLoader/assets/85258495/5ed81ea0-91f1-46fd-8b03-e7c8d0b7d705)

### Reproduction frequency

always

**Comment by JavidPack** [comment]
Which resource pack is causing this? I see 7 resource packs loading in your client.log, can you narrow down the issue to the specific pack and link it for us?

**Comment by QuibopWon** [comment]
quibtweakin
https://drive.google.com/file/d/1IjP-eSyP-BZT-FEW-p8NNcfURHOBEZnN/view?usp=share_link
can't embed it here cuz too big

**Comment by JavidPack** [comment]
Ok, so it also happens on Terraria. It isn't a bug, but an unexplained expectation of the image. 

Basically, it looks like vanilla underground backgrounds repeat themselves for the final 32 pixels along the width. Take the original Background_78 for example:

![image](https://github.com/tModLoader/tModLoader/assets/4522492/5a09deb7-e90b-4258-97a8-42b705224b00)

Your replacement texture does not follow this pattern.

![image](https://github.com/tModLoader/tModLoader/assets/4522492/6d32fb20-f79e-45b2-a7ee-b28454ccff19)

So basically, you'll need to adjust your images to account for the repetition that is expected.

**Comment by QuibopWon** [comment]
so i didn't need to manually make them tile
frick
